### DISCUSSION TITLE
GitHub package repository settings exposed to API

### DISCUSSION THREAD

**Discussion by trondtactile** [community/community#29643] [Actions]
Ever since new package repositories do not have `Inherit access from source repository` set by default, we are in need to set this value to true automatically whenever we create a new package repository. However, we can not find a way to do this through `api.github.com`.

Is there a way to achieve this? If not, could the API be updated to expose package repository settings?
Or better yet, can some default package repository settings be set on an organization level?

**Comment by Hobart2967** [comment]
+1

**Comment by cosst** [comment]
+1
